x-pack/plugins/security_solution/public/detections/components/host_isolation/use_sentinelone_host_isolation.tsx

```diff
@@ -20,27 +21,64 @@ interface ErrorType {
   meta: ActionTypeExecutorResult<SentinelOneGetAgentsResponse>;
 }
 
+// TODO:
+// 8.14
+// Remove this function when
+// agentStatusClientEnabled is enabled/removed
```

Comment:
any reason to write this TODO on 4 lines instead of 1?

Reply:
Not really. I can assimilate all these comments into a single line.

Reply:
97f6092864ff8adda824dc41ee2d28f3a23cd539